io.github.rocket.editor.yml

```
@@ -0,0 +1,33 @@
+app-id: io.github.rocket.editor
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: editor
+finish-args:
+  - --socket=wayland
```

```suggestion
  - --socket=wayland
  - --socket=fallback-x11
```
Or else it would not work under X11